common/country_types/hfe_country_types.txt

```diff
@@ -209,13 +249,25 @@ hfe_drone = {
 					base = 33
 				}
 			}
+			ancient_drone_station = {
+				min = 1
+				max = 14
+			}
+			ancient_mining_drone = {
+				min = 10
+				max = 200
+			}
 		}
 	}
 	faction = {
 		hostile = yes
+		hostile_when_attacked = yes
 		needs_border_access = no
 		generate_borders = yes
 		needs_colony = no
+		auto_delete = no
```

Comment:
Why no?

Reply:
The nation shouldn't up and destroy itself if it doesn't have a colony/ship/whatever/etc., and so on.

Reply:
If it have no ships/starbases, let it dissapear.

Reply:
Thoth is destroyed by Event, remember? If his station is destroyed, he gets a Game Over.

Reply:
Its not thoth

Reply:
Okay, done